OrbitClientServices/ProcessClient.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitClientServices/ProcessClient.h"
+
+#include "OrbitBase/Logging.h"
```

Comment:
Include what you use (unless it is in the header): `<utility>,<chrono>`

Reply:
Done, thanks!